From: Reinhard Tartler <siretart@tauware.de>
Subject: Use pkg-config for libpcsclite linkage flags

At least in debian, we can rely on pkg-config being available and
returning more accurate ldflags.
---
diff --git a/wpa_supplicant/Makefile b/wpa_supplicant/Makefile
index f3e86c1de6c0..ba3fe350675a 100644
--- a/wpa_supplicant/Makefile
+++ b/wpa_supplicant/Makefile
@@ -934,7 +934,7 @@ else
 ifdef CONFIG_OSX
 LIBS += -framework PCSC
 else
-LIBS += -lpcsclite -lpthread
+LIBS += $(shell $(PKG_CONFIG) --libs libpcsclite)
 endif
 endif
 endif
